video/out/vo_dmabuf_wayland.c

```diff
@@ -709,7 +710,7 @@ static int reconfig(struct vo *vo, struct mp_image *img)
     vo->target_params = &p->target_params;
     mp_mutex_unlock(&vo->params_mutex);
 
-    wl_surface_set_buffer_transform(vo->wl->video_surface, img->params.rotate / 90);
+    wl_surface_set_buffer_transform(vo->wl->video_surface, img->params.rotate / 90 + 4 * !!img->params.vflip);
```

Comment:
You can set VO_CAP_VFLIP capability for this vo, the compositor will handle the rotation here

Reply:
I thought I did, if it's missing it then that's a mistake.